tests/builder/axes_test.py

```diff
@@ -212,3 +215,51 @@ def test_mapping_is_same_regardless_of_axes_custom_parameter(ufo_module):
     assert doc.axes[0].minimum == 200
     assert doc.axes[0].maximum == 700
     assert doc.axes[0].map == []
+
+
+def test_custom_parameter_vfo_current():
+    """Tests get_regular_master when 'Variable Font Origin' custom parameter name
+    is used with master set to 'Regular Text'.  This is the current default
+    custom parameter name in the Glyphs editor / glyphs source file specification."""
+    source_path = os.path.join("tests", "data", "CustomParameterVFO.glyphs")
+    font = GSFont(source_path)
+    assert font.customParameters["Variation Font Origin"] is None
+    test_id = font.customParameters["Variable Font Origin"]
+    assert test_id == "ACC63F3E-1323-486A-94AF-B18797A154CE"
+    matched = False
+    for master in font.masters:
+        if master.id == test_id:
+            assert master.name == "Regular Text"
+            matched = True
+    assert matched is True
+    default_master = get_regular_master(font)
+    assert default_master.name == "Regular Text"
+
+
+def test_custom_parameter_vfo_old_name():
+    """Tests get_regular_master when 'Variation Font Origin' custom parameter name
+    is used with master set to 'Regular Text'.  This custom parameter name is not
+    used in current releases of the Glyphs editor / glyphs source file specification."""
+    source_path = os.path.join("tests", "data", "CustomParameterVFO-oldname.glyphs")
```

Comment:
I wonder if you could reduce the number of test files added to this PR. We don't need three of them probably, you can use one and modify the others (add or remove the custom parameter you need) before to call `get_default_master`

Reply:
3591e3f25624272ae5579482b6d43e20d201a4d8